.github/workflows/ci.yml

```diff
@@ -57,7 +57,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.7', '3.9']
+        python-version: ["3.7", "3.11"]
```

Comment:
How about including all supported versions?

Reply:
We do that on the full test routine that runs when the PR is merged. I think it is excessive to test against all Python versions every time there is an update in a PR